src/main/resources/changelog/changes/v4.0/create-reindex-entity-tables.xml

```diff
@@ -385,5 +387,125 @@
     <dropTable tableName="consortium_instance"/>
   </changeSet>
 
+  <changeSet id="MSEARCH-899@@create-sub_resources_locks-table" author="pavlo_smahin">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <tableExists tableName="sub_resources_lock"/>
+      </not>
+    </preConditions>
+
+    <comment>Create sub_resources_locks table</comment>
+
+    <createTable tableName="sub_resources_lock">
+      <column name="entity_type" type="VARCHAR(30)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_sub_resources_lock"/>
+      </column>
+      <column name="locked_flag" type="BOOLEAN">
+        <constraints nullable="false"/>
+      </column>
+      <column name="last_updated_date" type="DATETIME">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-899@@fill-sub_resources_lock-table" author="pavlo_smahin">
```

Comment:
Change the sequence of actions so that it would first add `last_updated_date` column to sub-resource table, and then create and fill `lock` tables.